File: src/projects/CoinFlipSim.java
package projects;

import java.util.Scanner;
import java.util.concurrent.ThreadLocalRandom;

public class CoinFlipSim {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Please enter a number of flips");
		int count = sc.nextInt();
		int heads = 0;
		int tails = 0;
		for(int i=0; i<count; i++) {
			int randomNum = ThreadLocalRandom.current().nextInt(1, 2 + 1);
			if(randomNum == 1) {
				heads++;
				System.out.println("Heads. Heads = "+heads+" | Tails = "+tails+" | Total Flips = "+(i+1));
			} else {
				tails++;
				System.out.println("Tails. Heads = "+heads+" | Tails = "+tails+" | Total Flips = "+(i+1));
			}
		}
		sc.close();
	}
}

File: src/projects/CoinFlipSim.py
import random

if __name__ == '__main__':
    flips = int(input("Please enter a number of flips\n"))
    heads = 0
    tails = 0
    for i in range(0, flips):
        randomNum = random.randrange(1,3)
        if randomNum == 1:
            heads += 1
            print("Heads. Heads = "+str(heads)+" | Tails = "+str(tails)+" | Total Flips = "+str(i+1))
        else:
            tails += 1
            print("Tails. Heads = "+str(heads)+" | Tails = "+str(tails)+" | Total Flips = "+str(i+1))
